src/apache/conf/ssl.conf

```diff
@@ -121,43 +121,18 @@ SSLRandomSeed connect file:/dev/urandom 512
     #   o OptRenegotiate:
     #     This enables optimized SSL connection renegotiation handling when SSL
     #     directives are used in per-directory context. 
-    <FilesMatch "\.(cgi|shtml|phtml|php)$">
+    <FilesMatch "\.php$">
         SSLOptions +StdEnvVars
     </FilesMatch>
-    <Directory "${SNAP}/cgi-bin">
-        SSLOptions +StdEnvVars
-    </Directory>
-
-    #   SSL Protocol Adjustments:
-    #   The safe and default but still SSL/TLS standard compliant shutdown
-    #   approach is that mod_ssl sends the close notify alert but doesn't wait for
-    #   the close notify alert from client. When you need a different shutdown
-    #   approach you can use one of the following variables:
-    #   o ssl-unclean-shutdown:
-    #     This forces an unclean shutdown when the connection is closed, i.e. no
-    #     SSL close notify alert is sent or allowed to be received.  This violates
-    #     the SSL/TLS standard but is needed for some brain-dead browsers. Use
-    #     this when you receive I/O errors because of the standard approach where
-    #     mod_ssl sends the close notify alert.
-    #   o ssl-accurate-shutdown:
-    #     This forces an accurate shutdown when the connection is closed, i.e. a
-    #     SSL close notify alert is send and mod_ssl waits for the close notify
-    #     alert of the client. This is 100% SSL/TLS standard compliant, but in
-    #     practice often causes hanging connections with brain-dead browsers. Use
-    #     this only for browsers where you know that their SSL implementation
-    #     works correctly. 
-    #   Notice: Most problems of broken clients are also related to the HTTP
-    #   keep-alive facility, so you usually additionally want to disable
-    #   keep-alive for those clients, too. Use variable "nokeepalive" for this.
-    #   Similarly, one has to force some clients to use HTTP/1.0 to workaround
-    #   their broken HTTP/1.1 implementation. Use variables "downgrade-1.0" and
-    #   "force-response-1.0" for this.
-    BrowserMatch "MSIE [2-5]" \
-        nokeepalive ssl-unclean-shutdown \
-        downgrade-1.0 force-response-1.0
 
     # Enable HSTS only if requested
     <IfDefine EnableHSTS>
         Header always set Strict-Transport-Security "max-age=63072000; includeSubdomains;"
     </IfDefine>
+
+	# Custom log for SSL to track ciphers used
+    CustomLog "${SNAP_DATA}/apache/logs/ssl_request_log" "%t %h %{SSL_PROTOCOL}x %{SSL_CIPHER}x \"%r\" %b"
```

Comment:
Log ciphers used.
It's really useful, but does a lot of writes and AFAIK we don't rotate logs.
Should we put it in the custom config?